RELEASE_NOTES.rst

```diff
@@ -42,7 +42,7 @@ coala 0.8.0 - grizzly
 For this release, we have had 46 developers from around the world contributing
```

Comment:
@amrandahvur I wouldn't change the release notes. They are of historic value and shouldn't be changed IMO.